docs/developing/integrate/wallets/cli.md

```diff
@@ -0,0 +1,101 @@
+---
+title: CLI
+proofedDate: 20241909
+iterationBy: na
+includedInSite: false
+approvedBy: na
+comment:
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+
+Get the Safe CLI
+Prerequisite: Install Docker Desktop (https://www.docker.com/products/docker-desktop/)
+
+Once Docker is installed on your system, create Safe account using the Safe guide
+
+## How to create a new Safe CLI
+
```

Comment:
The heading can be changed to "Interact with a Safe wallet using CLI"

Reply:
ready, new version is added